example/bayesian-methods/algos.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import print_function
 import mxnet as mx
 import mxnet.ndarray as nd
 import time
 import logging
 from utils import *
-
+from past.builtins import xrange
```

Comment:
$ python3
Python 3.4.3 (default, Nov 17 2016, 01:08:31) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import past
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'past'